SimPEG/Mesh/DiffOperators.py

```diff
@@ -565,7 +565,58 @@ def projNeumannOut(n, bc):
 
         return Pbc, Pin, Pout
 
-
+    
+    def unitCellGradx():
+        doc = """Cell centered Gradient in the x dimension used for 
+                regularization. The gradient operator is square (nC-by-nC)"""
+        def fget(self):
+            if self.dim < 3: return None
```

Comment:
This is why the tests are currently failing. The regularization tests 2D and 3D. We should be able to have a 1D, 2D, 3D unitCellGradx, right??